SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -314,7 +314,10 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	StaticComponentView = NewObject<USpatialStaticComponentView>();
 	SnapshotManager = NewObject<USnapshotManager>();
 	SpatialMetrics = NewObject<USpatialMetrics>();
-	VirtualWorkerTranslator = MakeUnique<SpatialVirtualWorkerTranslator>();
+	if (GetDefault<USpatialGDKSettings>()->bEnableUnrealLoadBalancer)
```

Comment:
late to the party, but this should also be guarded by IsServer().

Reply:
I see that change has already been made in master. This shouldn't cause errors if spawning on a client though, do you agree @yumnuska ?

Reply:
It shouldn't -- clients won't be able to do anything meaningful with the information.